packages/buildcop/src/buildcop.ts

```diff
@@ -278,40 +291,52 @@ buildcop.formatFailure = (failure: TestFailure): string => {
   return `${pkg}: ${failure.testCase} failed`;
 };
 
-buildcop.findFailures = (xml: string): TestFailure[] => {
-  const obj = xmljs.xml2js(xml);
-  const failures: TestFailure[] = [];
-  for (const suites of obj.elements) {
-    if (suites.name !== 'testsuites') {
-      continue;
+buildcop.findTestResults = (xml: string): TestResults => {
+  const obj = xmljs.xml2js(xml, { compact: true }) as xmljs.ElementCompact;
+  const failures: TestCase[] = [];
+  const passes: TestCase[] = [];
+  // Python doesn't always have a top-level testsuites element.
+  let testsuites = obj['testsuite'];
+  if (testsuites === undefined) {
+    testsuites = obj['testsuites']['testsuite'];
+  }
+  // If there is only one test suite, put it into an array to make it iterable.
+  if (!isIterable(testsuites)) {
```

Comment:
you can check `Array.isArray(testsuites)`, I'm sure there cannot be a real `Iterable` there.

Reply:
Perfect. Switched. Thanks!